utils/user-data-migration/index.js

```diff
@@ -0,0 +1,161 @@
+/*
+ * Migration script for user data provided by TopLevel, done in a rush
+ * Could be better structured
+ */
+
+const fileSystem = require('fs');
+const csvParse = require("csv-parse").parse;
+const { model: Model } = require('hof');
+const config = require('../../config');
+//@todo a more flexible way of passing in the the data file, the data file should not be part of the repo
+const userDataFilePath = __dirname + '/data-file/' + config.dataMigration.userDataFileName;
+const failedUserDataFilePath = __dirname + '/data-file/failedRows.csv';
+const keyCloakCreateUserEndpoint = config.keycloak.apiDomain + 'users'
+const { protocol, host, port } = config.saveService;
+const rdsUrl = `${protocol}://${host}:${port}/applicants`;
+const hofModel = new Model();
+const validator = require('hof').controller.validators;
+
+/**
+ * @param {int} applicantId
+ * @param {string} username
+ * @param {string} createdAt
+ * @param {string} updatedAt
+ * @returns {Object}
+ */
+const saveApplicantRecordToRdsService = async (applicantId, username, createdAt, updatedAt) => {
+    try {
+        const reqParams = {
+            url: rdsUrl,
+            method: 'POST',
+            data: {
+                applicant_id: applicantId,
+                username: username,
+                created_at: createdAt,
+                updated_at: updatedAt
+            }
+        };
+        const response = await hofModel._request(reqParams);
+        return response.data;
+    } catch (error) {
+        console.log(error);
+    }
+};
+
+/**
+ * @returns {Object}
+ */
+const getKeycloakAccessToken = async() => {
+    try {
+        const reqParams = {
+            url: config.keycloak.tokenUrl,
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/x-www-form-urlencoded'
+            },
+            data: {
+                client_id: config.keycloak.adminClient.clientId,
+                client_secret: config.keycloak.adminClient.secret,
+                grant_type: 'client_credentials'
+            }
+        };
+        const response = await hofModel._request(reqParams);
+        return response.data;
+
+    } catch (error) {
+        console.log(error);
+    }
+}
+
+/**
+ * @param {string} accessToken
+ * @param {string} username
+ * @param {string} email
+ * @param {string} createdAt
+ * @returns {Object}
+ */
+const createUserOnKeycloak = async (accessToken, username, email, createdAt) => {
+    //@todo check if the username do not exist first
+    try {
+        const reqParams = {
+            url: keyCloakCreateUserEndpoint,
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+                'Authorization': 'Bearer ' + accessToken
+            },
+            data: {
+                username: username,
+                enabled: true,
+                email: email,
+                emailVerified: true,
```

Comment:
Should be false, as we did not verify emails, and there is not confirmation that they have been verified
```suggestion
                emailVerified: false,
```